CONTRIBUTING.rst

```diff
@@ -0,0 +1,75 @@
+Contributing to this repository
+===============================
+
+|:smile:| |:heart_eyes:| First off, thanks for taking the time to contribute! With your help, we’re making developers' lives easier. |:heart_eyes:| |:smile:|
+
+The following is a set of tips and guidelines that will help you improve the Aiven developer portal.
+
+What should I know before I contribute?
+---------------------------------------
+
+**What is devportal?**
+Devportal is Aiven’s developer portal. It’s goal is to provide a comprehensive set of help articles aimed at developers using Aiven’s products. 
+
+**How does devportal work?**
+It is Python-based, with content in ReStructuredText (rst) and rendered by Sphinx.
+
+**What is the content model?**
+Devportal contains the following content types:
```

Comment:
(add a blank line between the paragraph and the list to make the bullet points render correctly)